index.html

```diff
@@ -2,377 +2,405 @@
-            </section>
-        <section class="about-me">
-            <div class="information">
-                <h3 class="aboutme-text">
-                    About me
-                </h3>
-                <p class="description-text"> I can help you build a product , feature or website Look through some of my
-                    work and experience! If you like what you
-                    see and have a project you need coded, don’t hestiate to contact me.</p>
-                <div class="get-mycv">
-                    <a href="#"><h6 class="btn-cv">Get my Resume </h6></a>
+              <div class="actions">
+                <div class="btn">
+                  <a href="#">
+                    <h6 class="btn-text">See Project</h6>
+                  </a>
                 </div>
+              </div>
             </div>
-            <div class="card">
-                <div class="language-card">
-                    <div class="lang-placeholder">
-                        <div class="title-tag">
-                            <h3 class="lang-title">Languages</h3>
-                            <ul class="tags">
-                                
-                                    <li class="tag-lang-name1"><h6 class="tag-language-name1"> JavaScript </h6></li>
-                                    <li class="tag-lang-name1"><h6 class="tag-language-name1">Ruby</h6> </li>
-                                    <li class="tag-lang-name1"><h6 class="tag-language-name1">html</h6> </li>
-                                    <li class="tag-lang-name1"><h6 class="tag-language-name1">CSS</h6></li>
-                                
-                            </ul>
-                        </div>
-                        <div class="icons">
-                            <div class="icon-languages">
-                                <div class="ellipse">
-                                    <div class="icon-langguage">
-                                        <div class="frame">
-                                           <a href=""> <img src="images/Lang.png" alt="language icon"/></a>
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-                </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="about-me">
+      <div class="information">
+        <h3 class="aboutme-text">
+          About me
+        </h3>
+        <p class="description-text"> I can help you build a product , feature or website Look through some of my
+          work and experience! If you like what you
+          see and have a project you need coded, don’t hestiate to contact me.</p>
+        <div class="get-mycv">
+          <a href="#">
+            <h6 class="btn-cv">Get my Resume </h6>
+          </a>
+        </div>
+      </div>
+      <div class="card">
```

Comment:
- [ ] Kindly ensure that the spacing between the technologies cards is even. You can use `display: flex` to justify them evenly.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/178084624-90e442f5-7526-4d94-a6c7-8f455aa1d957.png)  | ![image](https://user-images.githubusercontent.com/63473584/178084605-f2b9cd97-195b-478e-bcec-a63978c4cda9.png)  |